release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3356,6 +3356,70 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+* Previously, the installation program failed to install an {product-title} cluster in the `eu-es` (Madrid, Spain) region on a {ibm-power-server-title} platform that was configured as an `e980` system type. With this release, the installation program no longer fails to install a cluster in this environment. (link:https://issues.redhat.com/browse/OCPBUGS-38502[*OCPBUGS-38502*])
+
+* Previously, proxying for IDP communication occurred in the Konnectivity agent. By the time traffic reached Konnectivity, its protocol and hostname was no longer available. As a consequence, proxying was not done correctly for the OAUTH server pod. It did not distinguish between protocols that require proxying (http/s) and protocols that do not (ldap://). In addition, it did not honor the `no_proxy` variable that is configured in the `HostedCluster.spec.configuration.proxy` spec. With this release, you can configure the proxy on the Konnectivity sidecar of the OAUTH server so that traffic is routed appropriately, honoring your `no_proxy` settings. As a result, the OAUTH server can communicate properly with identity providers when a proxy is configured for the hosted cluster. (link:https://issues.redhat.com/browse/OCPBUGS-38058[*OCPBUGS-38058*])
+
+* Previously, if you created a hosted cluster by using a proxy for the purposes of making the cluster reach a control plane from a compute node, the compute node would be unavailable to the cluster. With this release, the proxy settings are updated for the node so that the node can use a proxy to successfully communicate with the control plane. (link:https://issues.redhat.com/browse/OCPBUGS-37937[*OCPBUGS-37937*])
+
+* Previously, the `iptables-alerter` pod did not handle errors from `crictl`, which could cause the pod to incorrectly log events from `host-network` pods or cause pod restarts. With this release, the errors are handled correctly so that these issues no longer persist. (link:https://issues.redhat.com/browse/OCPBUGS-37763[*OCPBUGS-37763*])
+
+* Previously introduced previously IPv6 support with UPI type installation caused an issue with naming OpenStack resources, which manifests itself on spinning up two UPI installation on the same OpenStack cloud. The outcome of this will set network, subnets and router happen to have the same name, which will interfere with one setup and prevent from deployment of the other. Now, all the names for mentioned resources will be unique per OpenShift deployment. (link:https://issues.redhat.com/browse/OCPBUGS-36855[*OCPBUGS-36855*])
```

Comment:
Nit: suggest an alternative to "spinning up" if possible, e.g.:
```suggestion
* Previously introduced previously IPv6 support with UPI type installation caused an issue with naming OpenStack resources, which manifests itself on starting two UPI installations on the same OpenStack cloud. The outcome of this will set network, subnets and router happen to have the same name, which will interfere with one setup and prevent from deployment of the other. Now, all the names for mentioned resources will be unique per OpenShift deployment. (link:https://issues.redhat.com/browse/OCPBUGS-36855[*OCPBUGS-36855*])
```

Reply:
Also removed extra "previously" :)